### ISSUE TITLE
[bug-website module] Mangadex chapters not populating

### ISSUE THREAD

**Issue by 7godeohs** [dazedcat19/FMD#1474]
### Prerequisites

- [X] I'm using the latest version.
- [X] All website modules are up to date.
- [X] This is not temporary server issue.

### System information

Windows 11 Pro 22H2

### Describe the bug

Downloads from Mangadex are not working for me.  If any title URL taken directly from their site is pasted and searched for within 'Manga Information -> Info' it retrieves cover art, but no chapters are populated.  Titles that have been added to Favorites are checked, but are highlighted red as though no content has loaded.

Affects all Mangadex-hosted chapters I have come across.

No issues with other sites (Mangasee for example) within FMD, including searching for titles, downloading specific chapters, or checking favorites.  Issues are specific to Mangdex only.

Issue only started occurring around 5-7 days ago.

Included 'downloads.db' in a zip file along with sample of Info window with cover art but no chapters listed.

### Log

[downloadsdb.zip](https://github.com/user-attachments/files/15529152/downloadsdb.zip)
![2024-06-03 01_37_14-Free Manga Downloader v2 0 32 0](https://github.com/dazedcat19/FMD2/assets/59428655/a2258324-e1a2-4857-9699-5a276a52c771)

**Comment by JayAyyBee** [comment]
Have you made sure your Mangadex language settings are to the language of your choice? In Options>Websites>Options.

**Comment by 7godeohs** [comment]
The language was actually set to Greek - had never changed it.  So I changed it to English, clicked 'Apply', then closed and reopened FMD.  No change in behavior.

I'm guessing it's not having an issue reaching Mangadex, because when it goes through the process of checking favorites, if a resource is totally unreachable, it pauses for a time on the title and shows an hourglass before moving on.  

In this case, there was no speed difference between checking Mangadex titles that remain highlighted red, and checking titles on sites where downloads work.

As part of troubleshooting before I commented, I also had downloaded the Mangadex.lua from GitHub and replaced the file that was on my computer.  So I should be up-to-date on that file.

**Comment by fuzzfactory0** [comment]
Just had this issue, checked the language as @JayAyyBee pointed out and found it was set to finnish. After setting it to english it works again. No idea how it got changed to finnish to begin with.

**Comment by 7godeohs** [comment]
It worked!

Very bizarre.  Like I said before, it had been set to Greek, I set it to English, 'Apply', and then restarted the app.  Still didn't work so I checked the settings and then closed it out.   Saw the response from @fuzzfactory0 and checked my language one more time.  It was set to Finnish - for whatever reason.  Changed it to English, 'Apply', and it's working now.  I'll know where to look in the future if this happens again.

Thanks to everyone in this community who creates, maintains, and supports this great program.  Just wanted to let you all know all this work and support is greatly appreciated.
